content/articles/dns-query-limits.md

```diff
@@ -17,53 +19,36 @@ TL;DR: We charge $0.10 per million queries per zone on Solo and Teams plans.
 
 ---
 
-## How we calculate fees
-
-- The price for up to, but not including, one million queries per zone is $0.10.
-- The DNS query volume fees are calculated at this price every time the registered volume goes beyond any given million-query mark (starting at 0 queries):
-    - The fee for 1 - 999,999 queries is $0.10
-    - The fee for 1M - 1,999,999 queries is $0.20
-    - The fee for 2M - 2,999,999 queries is $0.30
-    - And so on.
-- We won't charge fractions of the price:
-    - Any volume greater or equal to 1M and less than 2M will have a fee of $0.20
-    - For example, the fee for 1.5M queries is still $0.2.
-
-Let's explore this using a couple examples.
-
-### Example 1: 2,000,000 total queries across 4 zones
+## Calculating your fees
 
-This is a sample list of zones and queries for a billing period in an account:
-- zone1.com with 500,000 queries
-- zone2.com with 100,000 queries
-- zone3.com with 1,300,000 queries
-- zone4.com with 100,000 queries
+- 1 - 999,999 queries is $0.10
+- 1M - 1,999,999 queries is $0.20
+- 2M - 2,999,999 queries is $0.30
 
-In this scenario, you would see a DNS query volume fee of $0.50 in your invoice:
-- zone1.com, zone2.com, and zone4.com contribute each with $0.10, because they don't surpass the 1M queries threshold.
-- zone3.com contributes with $0.20, because it has registered between 1M and 2M queries.
+And so on.
 
-### Example 2: 2,000,000 total queries across a single zone
+We won't charge fractions of the price. Any volume greater or equal to 1M and less than 2M will cost $0.20.
 
-What if you have a single zone with a registered query volume of 2M queries?
-
-You would see a DNS query volume fee of $0.30 in your invoice, because zone1.com has registered a volume between 2M and 3M queries.
+**Try our pricing calculator (link or embed) to get an estimate of your zone query costs.**
```

Comment:
would love it if we could link straight to the calculator vs the entire pricing page, or add an embed (not sure that's possible here though)

Reply:
@dperrymorrow I think you posted a direct link at some point in Slack?

Reply:
Found it:

https://github.com/dnsimple/dnsimple-business/issues/1804#issuecomment-1957703190

Reply:
yeah, im not sure that we can embed the pricing calculator itself currently, but i could look into solutions for that if that is the direction we want to go.

Reply:
It's alright if we can't, I thought it would be fun if we could, but as long as we can link straight to it that's fine. I just didn't want to make people scroll through the pricing page to find it.